frontend/apps/picasso/pages/crowdloan-rewards/index.tsx

```diff
@@ -4,17 +4,21 @@ import { Box, Grid, Link, Typography, useTheme } from "@mui/material";
 import { useRouter } from "next/router";
 import Image from "next/image";
 import { useEffect } from "react";
-import { useStore } from "@/stores/root";
-import { FeaturedBox, PageTitle, SS8WalletHelper } from "@/components";
-import { ConnectorType, useConnector } from "bi-lib";
+import { PageTitle, FeaturedBox, SS8WalletHelper } from "@/components";
+import { ConnectorType, useBlockchainProvider, useConnector } from "bi-lib";
+import { useSelectedAccount } from "@/defi/polkadot/hooks";
 import { useDotSamaContext } from "substrate-react";
+import { useCrowdloanRewardsSlice } from "@/stores/defi/polkadot/crowdloanRewards/crowdloanRewards.slice";
+import { encodeAddress } from "@polkadot/util-crypto";
+import { SUBSTRATE_NETWORKS } from "@/defi/polkadot/Networks";
 
 const CrowdloanRewards: NextPage = () => {
   const theme = useTheme();
   const router = useRouter();
-  const userAssociation = useStore(
-    ({ crowdloanRewards }) => crowdloanRewards.associatedWith
-  );
+  const { account } = useBlockchainProvider(1);
```

Comment:
Possible to move this `1` outside and assign it to a constant with a better naming?